tests/lib.rs

```diff
@@ -1,3 +1,9 @@
+#[cfg(any(not(feature = "debug"), not(feature = "compare")))]
+compile_error!(
+    "Please enable the `debug` and `compare` features \
+                in order to compile and run the tests."
```

Comment:
might as well say how to enable the two features ?

Reply:
Yeah, maybe that's a good idea. You mean an example with `cargo`? E.g.

> Please enable the `debug` and `compare` features in order to compile and run the tests.
   Example: `cargo test --features debug --features compares`.

Reply:
That would do.